lms/validation/authentication/_launch_params.py

```diff
@@ -52,7 +56,15 @@ def lti_user(self):
         """
         kwargs = self.parse(locations=["form"])
 
-        return LTIUser(kwargs["user_id"], kwargs["oauth_consumer_key"], kwargs["roles"])
+        return LTIUser.from_names(
+            kwargs["lis_person_name_given"],
+            kwargs["lis_person_name_family"],
+            kwargs["lis_person_name_full"],
+            user_id=kwargs["user_id"],
+            oauth_consumer_key=kwargs["oauth_consumer_key"],
+            roles=kwargs["roles"],
+            tool_consumer_instance_guid=kwargs["tool_consumer_instance_guid"],
+        )
```

Comment:
So this is where we now generate the display name (`LaunchParamsAuthSchema` calls `from_names()`)